tests/test-cases/Group3-Docker-Compose/3-01-Docker-Compose-LEMP.robot

```diff
@@ -17,7 +17,7 @@ Compose LEMP Server
     Log To Console  \nThe VCH IP is %{VCH-IP}
 
     Run  cat %{GOPATH}/src/github.com/vmware/vic/demos/compose/webserving-app/docker-compose.yml | sed -e "s/192.168.60.130/${vch_ip}/g" > lemp-compose.yml
-    Run  cat lemp-compose.yml
+    ${output}=  Run And Return Rc And Output  cat lemp-compose.yml
```

Comment:
`Run And Return Rc And Output` returns the exit code and the output, so this should be:

`${rc}  ${output}=  Run And Return Rc And Output  cat lemp-compose.yml`

Also, if I'm understanding correctly, `cat lemp-compose.yml` is a no-op here since we're not inspecting its exit code and output. Should we remove this line altogether?